test/test_config/CMakeLists.txt

```diff
@@ -0,0 +1,61 @@
+find_package(Threads REQUIRED)
+
+add_executable(testconfig
+        $<TARGET_OBJECTS:libgerbera>
+        main.cc
+        test_configgenerator.cc
+        test_configmanager.cc
+        )
+
+include(DefFileName)
+define_file_path_for_sources(testconfig)
+
+include_directories(
+        ${UPNP_INCLUDE_DIRS}
+        ${UUID_INCLUDE_DIRS}
+        ${MAGIC_INCLUDE_DIRS}
+        ${ZLIB_INCLUDE_DIRS}
+        ${CURL_INCLUDE_DIRS}
+        ${LASTFMLIB_INCLUDE_DIRS}
+        ${FFMPEG_INCLUDE_DIR}
+        ${EXIF_INCLUDE_DIRS}
+        ${TAGLIB_INCLUDE_DIRS}
+        ${EXPAT_INCLUDE_DIRS}
+        ${FFMPEGTHUMBNAILER_INCLUDE_DIR}
+        ${DUKTAPE_INCLUDE_DIRS}
+        ${MYSQL_INCLUDE_DIRS}
+        ${SQLITE3_INCLUDE_DIRS}
+        ${ICONV_INCLUDE_DIR}
+        ${GTEST_INCLUDE_DIRS}
+)
+
+target_link_libraries(testconfig PRIVATE
+        ${UUID_LIBRARIES}
+        ${UPNP_LIBRARIES}
+        ${MAGIC_LIBRARIES}
+        ${ZLIB_LIBRARIES}
+        ${CURL_LIBRARIES}
+        ${LASTFMLIB_LIBRARIES}
+        ${FFMPEG_LIBRARIES}
+        ${EXIF_LIBRARIES}
+        ${TAGLIB_LIBRARIES}
+        ${EXPAT_LIBRARIES}
+        ${FFMPEGTHUMBNAILER_LIBRARIES}
+        ${DUKTAPE_LIBRARIES}
+        ${MYSQL_CLIENT_LIBS}
+        ${SQLITE3_LIBRARIES}
+        ${ICONV_LIBRARIES}
+        ${GTEST_LIBRARIES}
+        ${GERBERA_INTERFACE_LIBRARIES}
+        ${CMAKE_THREAD_LIBS_INIT}
+        )
+
+add_custom_command(TARGET testconfig POST_BUILD
+        COMMAND ${CMAKE_COMMAND} -E copy_directory ${CMAKE_CURRENT_SOURCE_DIR}/fixtures $<TARGET_FILE_DIR:testconfig>/fixtures
```

Comment:
Copy all the **fixture** files to the build directory, thus allowing easy compare within GoogleTest